HousingSearchApi.Tests/V1/E2ETests/Fixtures/BaseFixture.cs

```diff
@@ -58,5 +62,37 @@ protected void EnsureEnvVarConfigured(string name, string defaultValue)
                 _elasticSearchAddress = default;
             }
         }
+
+        protected static List<List<RelatedEntity>> GenerateRelatedEntities(int count)
```

Comment:
I think this should go into the process fixture as it's specifically for processes, not any other data entity